carbon-design-system/carbon-addons-iot-react
# feat(image-hotspots): add minimapBehavior prop for better control

Closes #1356 

**Summary**

- adds a `minimapBehavior` prop for better control over when the minimap is displayed.

**Change List (commits, features, bugs, etc)**

- Add `minimapBehavior` prop
- Deprecate `hideMinimap` prop
- Add tests to confirm behavior is as expected
- fixed styles on minimize icon when zoomed in.
- updated snapshots

**Acceptance Test (how to verify the PR)**

- in the [https://deploy-preview-2345--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-imagehotspots--landscape-image-landscape-container story](https://deploy-preview-2345--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-imagehotspots--landscape-image-landscape-container)
  - zooming in and panning should show the minimap by default.
  - change the minimapBehavior prop to 'hide' and the minimap should always be hidden. even when panning.
  - change the minimapBehavior prop to 'show' and the minimap should always be visible--even when at default zoom or when panning.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- previously expected behavior (only showing minimap when panning) should be seen in all stories
